React/tic-tac-toe/src/app/screens/Login/layout.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+
+import CustomInput from '../../components/Input';
+
+import { validate } from './validation';
+
+class Form extends Component {
+  render() {
+    /* eslint-disable */
+    const { handleSubmit } = this.props;
+    /* eslint-enable */
```

Comment:
Why do you need to disable eslint?
Let's see the error, but if you ever need to disable something add the rule name.